EIPS/ethereum-access-token-eip.md

```diff
@@ -0,0 +1,188 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7272
```
Assigning EIP number using PR number.  Please also update the filename.

Reply:
Done 👍